week2/homework/src/index.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+const fs = require('fs');
+
+let fileName = 'toDo.json';
+// console.log(fileName);
+function readFile() {
+    return new Promise(
+        resolve => fs.readFile(
+            fileName,
+            (err, data) => resolve(err ? '' : data.toString())
+        )
+    );
+}
+
+function remove() {
+    readFile()
+        .then(content => {
+            let newArray = content.split('\n');
+            let i = newArray.length-1
+            newArray.splice(i, 1);
```

Comment:
You are always removing the last element of the array. At line 60 you call the `remove` function with `removeArgs` but it is not used here.